src/main/scala/strawman/collection/Iterable.scala

```diff
@@ -206,6 +206,12 @@ trait IterableOps[+A, +CC[X], +C] extends Any {
     drop(1)
   }
 
+  /** The initial part of the collection without its last element. */
+  def init: C = {
+    if (coll.isEmpty) throw new UnsupportedOperationException
+    take(size - 1)
```

Comment:
Without `dropRight`, this is the canonical implementation.  However, with `dropRight`, I think this should be `dropRight(1)`.  Maybe leave a comment here to prompt revisiting this once we have `dropRight`?